arviz/utils/xarray_utils.py

```diff
@@ -277,6 +336,7 @@ def to_inference_data(self):
             'sample_stats': self.sample_stats_to_xarray(),
             'posterior_predictive': self.posterior_predictive_to_xarray(),
             'prior': self.prior_to_xarray(),
+            'data': self.data_to_xarray(),
```

Comment:
The only thing that concerns me about this change is that we use the variable data throughout the library to refer to  posterior. 

Would you alright if I opened an issue and/or made a pull request to rename this in the various plots? For example traceplot

https://github.com/arviz-devs/arviz/blob/master/arviz/plots/traceplot.py

Reply:
I don't like it either. `data` is really arbitrary thing.

Could it be `observed`, `observed_data` or something else?

Reply:
I think calling it `data` was a nod to stan's data section :)

I updated it to 'observed_data', and also updated `schema.md`. @canyon289 let me know if that's ok with you, too.